org.eclipse.jdt.ls.tests/src/org/eclipse/jdt/ls/core/internal/correction/AbstractQuickFixTest.java

```diff
@@ -179,6 +190,22 @@ protected void setIgnoredCommands(List<String> ignoredCommands) {
 		parms.setContext(context);
 
 		List<Either<Command, CodeAction>> codeActions = new CodeActionHandler(this.preferenceManager).getCodeActionCommands(parms, new NullProgressMonitor());
+
+		if (this.ignoredKinds != null) {
+			List<Either<Command, CodeAction>> filteredList = new ArrayList<>();
```

Comment:
filteredList = codeActions.stream().filter(Either::isRight).filter(codeAction -> {
for (String str : this.ignoredKinds) {
	if (codeAction.getRight().getKind().matches(str)) {
			return false;
	}
}
return true;
}).collect(Collectors.toList())